perk-store-webapps/src/main/webapp/perkstoreOrder.html

```diff
@@ -5,5 +5,30 @@
                 perkstoreOrderApp.init();
             })
         </script>
+        <div data-app="true" class="v-application v-application--is-ltr theme--light" id="perkstoreOrderPortlet" flat="">
+            <div class="v-application--wrap">
+                <main>
+                    <div class="container pa-0">
+                        <div class="layout row wrap mx-0" style="cursor: pointer;">
+                            <div class="flex d-flex sx12">
+                                <div class="layout white row ma-0">
+                                    <div class="flex d-flex xs12">
+                                        <div class="v-card v-card--flat v-sheet theme--light">
+                                            <div class="v-card__text subtitle-2 grey--text pa-2"><span class="skeleton-text skeleton-background skeleton-header skeleton-border-radius">
+                              Mes commandes
```

Comment:
Replace this litteral translated string by `&nbsp;`

Reply:
Done!